src/Lifi.ts

```diff
@@ -213,15 +265,29 @@ class LIFI {
     }
   }
 
+  /**
+   * Get a list of active routes.
+   * @return {Route[]} A list of routes.
+   */
   getActiveRoutes = (): Route[] => {
     return Object.values(this.activeRoutes).map((dict) => dict.route)
   }
 
+  /**
+   * Get a single updated route. The route has to be active.
+   * @param {Route} route - A route object.
+   * @return {ReturnValueDataTypeHere} Brief description of the returning value here.
```

Comment:
Please add a "Brief description of the returning value here"

Reply:
ohh my bad